mkt/extensions/validation.py

```diff
@@ -70,10 +79,13 @@ def __init__(self, file_obj=None):
         self.file_obj = file_obj
         self.zipfile = None
 
-    def error(self, error_key):
+    def error(self, error_key, error_parameter=None):
+        message = self.errors[error_key]
+        if error_parameter is not None:
+            message = self.errors[error_key] % error_parameter
         raise ParseError(detail={
             'key': error_key,
-            'message': self.errors[error_key],
+            'message': message,
         })
```

Comment:
If it's parameterized, can we return all the parameters in the response as well? That might make construction of a [more user-friendly error message](https://github.com/mozilla/marketplace-content-tools/blob/master/src/media/js/addon/components/validationError.js) a little easier.